<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1458</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1401</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab_0">
     <attribute name="title">
      <string>航班</string>
     </attribute>
     <widget class="QPushButton" name="addFlightButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>450</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteFlightButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>450</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_0">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>1051</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>450</y>
        <width>151</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>show Report</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>航班查询</string>
     </attribute>
     <widget class="QTableView" name="tableView_1">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>60</y>
        <width>931</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="postButton">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>420</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>提交订单</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>291</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <widget class="QTimeEdit" name="timeEdit"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="FFROM_lineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>起飞日期</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="FTO_lineEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>起飞 时间</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>航空公司</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="queryButton">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="FLIGHT_comboBox">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>到达地</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>出发地</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Order By</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="order_comboBox">
         <item>
          <property name="text">
           <string>最近</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>价格</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDateEdit" name="dateEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>450</y>
        <width>93</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>是否中转</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>480</y>
        <width>201</width>
        <height>29</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string> 中转时间</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="transtime_lineEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>订单管理</string>
     </attribute>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>861</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="checkin_pushButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>450</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>check-in</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Unsubscribe_pushButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>450</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Unsubscribe</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>账单查询</string>
     </attribute>
     <widget class="QTableView" name="tableView_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>771</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>241</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>The Recent Flight</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>报表</string>
     </attribute>
     <widget class="QComboBox" name="FLIGHT_comboBox_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>450</y>
        <width>219</width>
        <height>27</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>731</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1458</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>航班信息(&amp;F)</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_F"/>
   </widget>
   <widget class="QMenu" name="menu_O">
    <property name="title">
     <string>订单管理(&amp;O)</string>
    </property>
    <addaction name="action_Q"/>
    <addaction name="action_L"/>
    <addaction name="action_G"/>
    <addaction name="action_B"/>
   </widget>
   <widget class="QMenu" name="menu_T">
    <property name="title">
     <string>账单查询(&amp;T)</string>
    </property>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_O"/>
   <addaction name="menu_T"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_F">
   <property name="text">
    <string>航班信息(&amp;F)</string>
   </property>
  </action>
  <action name="action_Q">
   <property name="text">
    <string>航班查询(&amp;Q)</string>
   </property>
  </action>
  <action name="action_G">
   <property name="text">
    <string>取票(&amp;G)</string>
   </property>
  </action>
  <action name="action_B">
   <property name="text">
    <string>退订机票(&amp;B)</string>
   </property>
  </action>
  <action name="action_L">
   <property name="text">
    <string>订单查询(&amp;L)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>FFROM_lineEdit</tabstop>
  <tabstop>FTO_lineEdit</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>timeEdit</tabstop>
  <tabstop>FLIGHT_comboBox</tabstop>
  <tabstop>order_comboBox</tabstop>
  <tabstop>queryButton</tabstop>
  <tabstop>postButton</tabstop>
  <tabstop>tableView_0</tabstop>
  <tabstop>addFlightButton</tabstop>
  <tabstop>deleteFlightButton</tabstop>
  <tabstop>tableView_2</tabstop>
  <tabstop>tableView_1</tabstop>
  <tabstop>checkin_pushButton</tabstop>
  <tabstop>Unsubscribe_pushButton</tabstop>
  <tabstop>tableView_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
